### ISSUE TITLE
Compatibility with CUDA 7.5RC?

### ISSUE THREAD

**Issue by oscarbg** [NVlabs/SASSI#3]
Hi,
just found the project and would like to know if CUDA 7.5 will require new releases of SASSI (and in general every CUDA release) or not?
thanks..

**Comment by mwstephenson** [comment]
You _might_ be able to use SASSI with CUDA 7.5.  The SASSI installer simply duplicates an existing CUDA installation, installs some additional header files, and replaces the PTX assembler (ptxas) with the SASSI-fied version.  I have personally tried using older versions of CUDA with the CUDA 7 version of SASSI, and it has worked.  But it may not be forward compatible. Officially "supporting" CUDA 7.5 will require a new SASSI release.  Porting SASSI to CUDA 7.5 will not be hard, but I won't start on it until it moves beyond RC.

Thanks!
Mark

**Comment by pr0metha** [comment]
Hi,
What is the current status of the 7.5 port? I have tried to use CUDA-7.5 together with SASSI, but it doesn't work due to the fact that ptx v4.3 is unsupported by the SASSI-fied ptxas. I would be really happy if I could use SASSI together with half-precision floats.

Thanks!

**Comment by mwstephenson** [comment]
Hi there, my tentative plan is to target a CUDA 8 port.  It will work for Fermi, Kepler, and Maxwell, but it will not work in the short term for Pascal.

I agree, it would be cool to profile fp16!

Thanks,
Mark

**Comment by sderek** [comment]
Hi,

Just wondering any updates for CUDA 7.5 or 8 ?

thanks,

**Comment by 1Husin-Alhaj** [comment]
SASSI is just compatible with CUDA 7. If you configure sassi with non-cuda 7, you will get an error during compiling files. Error message is like this: unsupporting ptx versin 4.3, only version 3.2. I faced this error. Then I removed cuda 7.5, and installed cuda 7, and CUDA worked correctly.

**Comment by mwstephenson** [comment]
Yes, that's correct.  SASSI is restricted to CUDA 7.  Sorry for the
inconvenience.

-Mark
